src/main/java/net/backupcup/mcde/screen/util/EnchantmentSlotsRenderer.java

```diff
@@ -75,27 +78,28 @@ private EnchantmentSlotsRenderer(
         this.dimPredicate = dimPredicate;
         this.defaultGuiTexture = defaultGuiTexture;
         this.dimColorMultiplier = dimColorMultiplier;
+        this.resourceManager = resourceManager;
     }
 
-    public void drawSlot(MatrixStack matrices, Slots slot) {
+    public void drawSlot(MatrixStack matrices, SlotPosition slot) {
```

Comment:
There is a difference between `SlotPosition` and `Slot`. This should be reflected in your variable's name. Therefore, consider `slotPosition` over `slot`.